### PR TITLE
Updates development docs

### PR DESCRIPTION
**Pull request by PuneetPunamiya**

### CODE REVIEW

File: docs/DEVELOPEMENT.md [hunk 1]
```diff
@@ -13,54 +13,55 @@ NOTE:- You need to install [tektoncd/pipeline](https://github.com/tektoncd/pipel
...
-4. Now approve/reject the approval task using curl request
 
-    ```shell
-    curl --header "Content-Type: application/json" \                                                                                           
-    --request POST \
-    --data '{"approved":"false", "namespace":"default"}' \
-    http://localhost:8000/approvaltask/<approvalTaskName>
-    ```
\ No newline at end of file
+3. Now `approve/reject` the approval task by using `kubectl edit` command
```
Comment (rupalibehera) [inline docs/DEVELOPEMENT.md:67]: `kubectl edit approvaltask <taskname> --as=<user>` lets also mention an example command

File: docs/DEVELOPEMENT.md [hunk 2]
```diff
@@ -13,54 +13,55 @@ NOTE:- You need to install [tektoncd/pipeline](https://github.com/tektoncd/pipel
...
-               steps:
-                 - image: busybox
-                   name: after
-                   script: echo after wait
-             runAfter: ['wait']
-    
+      apiVersion: tekton.dev/v1
+      kind: PipelineRun
+      metadata:
+      generateName: test-
```
Comment (rupalibehera) [inline docs/DEVELOPEMENT.md:22]: indentation error

File: docs/DEVELOPEMENT.md [hunk 3]
```diff
@@ -13,54 +13,55 @@ NOTE:- You need to install [tektoncd/pipeline](https://github.com/tektoncd/pipel
...
-    go run ./cmd/approver/main.go
-    ```
-4. Now approve/reject the approval task using curl request
 
-    ```shell
-    curl --header "Content-Type: application/json" \                                                                                           
-    --request POST \
-    --data '{"approved":"false", "namespace":"default"}' \
-    http://localhost:8000/approvaltask/<approvalTaskName>
-    ```
```
[the remark this answers is not in the document]
Comment (PuneetPunamiya) [reply docs/DEVELOPEMENT.md:66]: updated!

**Review by rupalibehera — APPROVED** [review]
LGTM

**Pull request merged**